pkgs/applications/networking/instant-messengers/dino/default.nix

```diff
@@ -40,37 +37,28 @@ stdenv.mkDerivation rec {
   buildInputs = [
     qrencode
     gobject-introspection
-    glib-networking
     glib
+    glib-networking # required for TLS support
+    libadwaita
     libgee
-    gnome.adwaita-icon-theme
     sqlite
     gdk-pixbuf
-    gtk3
+    gtk4
     libnotify
     gpgme
     libgcrypt
     libsoup
-    pcre
```

Comment:
this got replaced with pcre2 but I only saw pkg-config complain that it is missing. It is not used in code.

Reply:
Hmm, should we ask upstream about this one? (PR to remove the pkg-config check, maybe?)

Reply:
It is not a hard config check but pkg-config spits some errors out which don't make the build fail. I'd rather left it then trying to figure out what would break by removing it. There are other warnings related to glib dependencies, too.